content/pt/application_programming_interface.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Interface de programação de aplicações
+status: Completed
+category: technology
+---
+
+## O que é
+
+Uma API (interface de programação de aplicações - do inglês: applications protocol interface) é um modo como programas de computador interagem uns com os outros. Assim como humanos interagem com um site através de uma página web, uma API permite que programas de computador interajam uns com os outros. Ao contrário das interações humanas, as APIs possuem limitações sobre o que pode e o que não pode ser solicitado a elas. Essa interação limitada ajuda a criar uma comunicação estável e funcional entre programas.
+
+## Qual problema é resolvido
+
+À medida que as aplicações se tornam mais complexas, pequenas mudanças no código podem ter efeitos drásticos em outras funcionalidades. As aplicações precisam adotar uma abordagem modular para a sua funcionalidade se puderem crescer e escalar simultaneamente. Sem as APIs, existe uma falta de um framework para a interação entre as aplicações. Sem um framework compartilhado, é desafiador escalar e integrar as aplicações.
+
+## No que isso ajuda
+
+APIs permitem que programas de computador ou aplicações interajam e compartilhem informações de uma maneira definida e compreensível. Elas são os blocos de construção para as aplicações modernas e fornecem aos desenvolvedores uma maneira de integrar aplicações. Sempre que você ouvir sobre [microsserviços](/content/en/microservices.md) você pode inferir que eles interagem uns com os outros por meio de uma API.
```

Comment:
for now I linked the english page for microsservices, since we did not translated it yet to portuguese

Reply:
Sounds good